mataa_tools/mataa_ESS_sync.m

```diff
@@ -0,0 +1,117 @@
+function [h, t] = mataa_ESS_sync(P, T, fs, latency, cal)
```

Comment:
I'd suggest to change the function name following the logic of the other MATAA functions. This is a measurement, so it should be mataa_measure_XXX. The measurement result is an impulse response (IR) and the harmonic distortion (HD). How about mataa_measure_IR_HD?